lib/probe_dock_ruby/annotation.rb

```diff
@@ -0,0 +1,86 @@
+module ProbeDockProbe
+  class Annotation
+    BASE_REGEXP_STRING = '@probedock\(([^\(\)]*)\)'
+    ANNOTATION_REGEXP = /#{BASE_REGEXP_STRING}/
+    STRIP_ANNOTATION_REGEXP = /\s*#{BASE_REGEXP_STRING}/
+
+    attr_reader :key, :category, :tags, :tickets, :active
+
+    def initialize(str)
+      parse(str)
+    end
+
+    def merge(annotation)
```

Comment:
It's very un-Ruby like to call this `merge`. It should be `merge!`, or it should return a new object.